### ISSUE TITLE
Some issues related to history_map

### ISSUE THREAD

**Issue by GuoxingYan** [TonyXuQAQ/RNGDetPlusPlus#12]
By making the dataset code, the generated dataset, whether it's mask or point, has more than one pixel width,
However, the pixel width of history_map is 1. Is this normal?

**Comment by TonyXuQAQ** [comment]
history_map is a raster binary map. Except the ground truth vertices in the next step (i.e, $V^*$), other labels for training are raster masks. Please wait for the training code.

**Comment by TonyXuQAQ** [comment]
The sampling and training codes are released. Please refer to the implementation code. This issue is closed.
